src/lib/parse/blueprint.js

```diff
@@ -69,9 +68,20 @@ module.exports = function(filePath: string, autoOptions: boolean, routeMap: {},
                 var parsedUrl = urlParser.parse(resource.uriTemplate);
                 var key = parsedUrl.url;
                 routeMap[key] = routeMap[key] || { urlExpression: key, methods: {} };
-                parseParameters(parsedUrl, resource.parameters, routeMap);
+                const queryParamKeys = Object.keys(parsedUrl.queryParams);
```

Comment:
Perhaps replace the loss of `parseParameters` clarifying method with a comment `Extract query parameters and path parameters`